README.md

```diff
@@ -9,7 +9,7 @@
 ## Documentation
 The W2 Documents API consits of 3 node projects:
 - The `/API` that handles the connection to W2 server and converts documents. 
-- `/authorizer`
+- `/authorizer` that relies on the [node-lambda-authorizer](https://github.com/LBHackney-IT/node-lambda-authorizer).
```

Comment:
looks good. please merge master in - there have been some additions to the readme